webpack.common.config.js

```diff
@@ -110,6 +118,9 @@ var resolve = {
         'koHelpers': staticPath('js/koHelpers.js'),
         'addonPermissions': staticPath('js/addonPermissions.js'),
         'navbar-control': staticPath('js/navbarControl.js'),
+        'osf-panel': staticPath('vendor/bower_components/osf-panel/dist/jquery-osfPanel.js'),
```

Comment:
This should not be necessary. Since `osf-panel` is bower-installed and has a `main` entry, `require('osf-panel')` will do the right thing.